rex-release/out.sh

```diff
@@ -47,6 +57,13 @@ upload-release() {
 
   # books files with explicit content type set because they don't have extensions, loaded unversioned so no cloudfront caching
   aws s3 sync --exclude 'service-worker.js' --content-type 'text/html' --cache-control 'max-age=0' "$path/books/" "s3://$bucket/rex/releases/$version/books"
+
+  # configure redirects
+  for row in $(jq -c '.[]' < $path/rex/redirects.json); do
+    from=$(jq -r '.from' <<< "$row")
+    to=$(jq -r '.to' <<< "$row")
+    aws s3api put-object --bucket "$bucket" --key "rex/releases/$version$from" --website-redirect-location "$to"
```

Comment:
can we find some docs that verify this is intended to generate 301 redirects?

Reply:
It [looks](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-page-redirect.html) like `x-amz-website-redirect-location` is used for 301 redirects by default, but if we would need to add some redirect rules using Rest api, we could use [PutBucketWebsite](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketWebsite.html).